src/instructlab/server.py

```diff
@@ -68,11 +69,11 @@ def ensure_server(
             tls_client_key=tls_client_key,
             tls_client_passwd=tls_client_passwd,
         )
-        return (None, None)
+        return (None, None, None)
     except ClientException:
         tried_ports = set()
-        # TODO: use default server, "spawn" doesn't work?
-        mpctx = multiprocessing.get_context(None)
+        # TODO: use fork, "spawn" fails with semaphore FileNotFound on Linux.
+        mpctx = multiprocessing.get_context("fork" if sys.platform == "linux" else None)
```

Comment:
Can you try with new default here, too?

Reply:
I just pushed a change to use `None` everywhere. Let's see what happens!

Reply:
It passed the first full run. I kicked it to run again to be sure.